src/http.c

```diff
@@ -279,7 +279,17 @@ int mg_http_parse(const char *s, size_t len, struct mg_http_message *hm) {
   hm->uri.buf = (char *) s;
   while (s < end && (n = clen(s, end)) > 0) s += n, hm->uri.len += n;
   while (s < end && s[0] == ' ') s++;  // Skip spaces
+  is_response = mg_ncasecmp(hm->method.buf, "HTTP/", 5) == 0;
   if ((s = skiptorn(s, end, &hm->proto)) == NULL) return false;
+  // If we're given a version, check that it is HTTP/x.x
+  bool version_prefix_valid = mg_ncasecmp(hm->proto.buf, "HTTP/", 5) == 0;
```

Comment:
Keep this code C89 compliant (e.g. make -C test vc98 should pass): can't declare a var in the middle of the block